packages/core/rollup.config.js

```diff
@@ -48,9 +49,11 @@ const createUmdConfig = ({ input, output, target = undefined }) => ({
   ]
 });
 
+const npmInputs = glob.sync('src/!(scxml|invoke|model.types|typegenTypes).ts');
```

Comment:
those were mainly type-only files and Rollup has been complaining about empty entries so I've filtered them out. The whole thing should still work correctly - and those files are still accessible at the type-level because `.d.ts` files are still produced for them